model/components/sentry.yml

```diff
@@ -0,0 +1,43 @@
+---
+components:
+
+  # sentry
+  - id: sentry
+    name: Sentry
+    description: Sentry is cross-platform application monitoring, with a focus on error reporting.
+    link: https://sentry.io
+    logo: sentry
+    license: BSD 3-Clause "New" or "Revised" License
+    categories:
+      - error-reporting
```

Comment:
do we really need a new category? is `aspect` not enough.. I am not sure, never heard of Sentry. But at the first look this looks more like an `instrumentation-lib` category for me.. @AlexanderWert ideas?